app/assets/images/symbol-defs.svg

```diff
@@ -354,8 +31,5 @@
 </symbol>
```

Comment:
These 3 don't have corresponding css font icons.

The rest of them that I kept (the checkmarks and crosses etc) remain because we use them as SVGs in some places. For the ones I deleted, I don't _think_ we're using them anywhere as SVGs, but I could be missing something.